From b4bd90312b9cf486af0ad4babc308cf5c19e4892 Mon Sep 17 00:00:00 2001
From: Andre McCurdy <armccurdy@gmail.com>
Date: Thu, 12 Nov 2015 15:40:00 -0800
Subject: [PATCH] nexus/build/os/linuxuser/os_flags.inc: always set
 B_REFSW_TOOLCHAIN_VER_GE_4_8

The current test (ie searching for the string '4.8.') fails for gcc 4.9,
5.2, etc. Since all current OE toolchains are >= gcc 4.8, just hardcode
B_REFSW_TOOLCHAIN_VER_GE_4_8 to 'y'.

Signed-off-by: Andre McCurdy <armccurdy@gmail.com>
---
 nexus/build/os/linuxuser/os_flags.inc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/nexus/build/os/linuxuser/os_flags.inc b/nexus/build/os/linuxuser/os_flags.inc
index 3d7440b..30bbe56 100644
--- a/nexus/build/os/linuxuser/os_flags.inc
+++ b/nexus/build/os/linuxuser/os_flags.inc
@@ -68,11 +68,8 @@ IS_TOOLCHAIN_VALID := y
 endif
 ifeq ($(IS_TOOLCHAIN_VALID),y)
 B_REFSW_TOOLCHAIN_VER := $(shell $(B_REFSW_CROSS_COMPILE)gcc -dumpversion)
-# TODO: parse string for true "greater than or equal" test
-ifneq ($(findstring 4.8.,$(B_REFSW_TOOLCHAIN_VER)),)
 B_REFSW_TOOLCHAIN_VER_GE_4_8 := y
 endif
-endif
 
 # We use c89 for maximum compatibility across OS's
 # But, we allow overrides for integrating other code
-- 
1.9.1

